### ISSUE TITLE
Impossible to use --credentials parameter

### ISSUE THREAD

**Issue by aborruso** [jsvine/twick#3]
Hi,
I have tried to us twick in this way:
twick fetch "harlem building collapse" --credentials qIDzHFUqeasPJiOkN4rjR1111,M6VAzW0bi7VS4iSHHN6tuLigTXoD85jWmhEJi6vUSEnj0v1111,1317611-7VVW1lcceni3MtDNkmnDtkumzRokWCxeYQ9xTK1111,pgU3nhsYSgdpCePZbJUBH0gOT49ZOPPmBnzhGgIYs1111 --db sqlite:///tweets3.db

But I have this error:
except: raise Exception("Missing environment variable ${}".format(name))
Exception: Missing environment variable $TWICK_API_KEY

It seems that my twick  command does not read  --credentials parameter. What's wrong in my command?

Thank you

**Comment by jsvine** [comment]
Hi @aborruso, I have a hunch about what might be wrong. `--credentials` requires the credentials to be space-separated, and it looks like your command is comma-separated instead. If you try this, does it work?:

```sh
twick fetch "harlem building collapse" --credentials qIDzHFUqeasPJiOkN4rjR1111 M6VAzW0bi7VS4iSHHN6tuLigTXoD85jWmhEJi6vUSEnj0v1111 1317611-7VVW1lcceni3MtDNkmnDtkumzRokWCxeYQ9xTK1111 pgU3nhsYSgdpCePZbJUBH0gOT49ZOPPmBnzhGgIYs1111 --db sqlite:///tweets3.db
```

**Comment by aborruso** [comment]
Hi @jsvine,
I have tried and I have always 
"except: raise Exception("Missing environment variable ${}".format(name))
Exception: Missing environment variable $TWICK_API_KEY"

Thank you

**Comment by jsvine** [comment]
My apologies @aborruso. There was a bug in the credential-reading code. I've fixed that, and pushed the changes to v0.1.0. Run `pip install --upgrade twick`, and then retry your command. If that doesn't work, let me know. Many thanks for flagging this issue!

**Comment by aborruso** [comment]
it works, thank you
